django/contrib/admin/templates/admin/submit_line.html

```diff
@@ -9,5 +9,8 @@
 {% if show_save_as_new %}<input type="submit" value="{% trans 'Save as new' %}" name="_saveasnew">{% endif %}
 {% if show_save_and_add_another %}<input type="submit" value="{% trans 'Save and add another' %}" name="_addanother">{% endif %}
 {% if show_save_and_continue %}<input type="submit" value="{% trans 'Save and continue editing' %}" name="_continue">{% endif %}
+{% if show_close %}
+  <a href="{% url opts|admin_urlname:'changelist' %}" class="closelink">{% trans 'Close without saving' %}</a>
```

Comment:
Will this render as "Close without saving" if we only have view permissions and this cannot save anyway?

Reply:
Yes, I think it's better to be clear that there's no saving going on despite clicking a button that's located where the close buttons usually are.

I think it stays quite clear for the user, because that button only appears if there's no editable form on the page anyways.